src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -122,6 +122,17 @@ public function load(array $configs, ContainerBuilder $container)
             $twigFilesystemLoaderDefinition->addMethodCall('addPath', array($dir));
         }
 
+        if ($container->fileExists($dir = $container->getParameter('kernel.project_dir').'/templates', false)) {
+            $twigFilesystemLoaderDefinition->addMethodCall('addPath', array($dir));
+        }
+
+        if ($container->fileExists($dir = $container->getParameter('kernel.project_dir').'/templates/bundles', false)) {
+            foreach (glob($dir.'/*', GLOB_ONLYDIR) as $path) {
```

Comment:
I'm not sure to get your comment right. Why would we add paths that do not exist? Here we just want to add paths for existing folders that need to override bundles at the application level.

How would you see it?